examples/mapswipe.css

```diff
@@ -3,6 +3,30 @@
     height: 400px;
   }
 
-.swipe-input {
+ngeo-mapswipe .swipe-input {
+  background: none;
+  height: 0;
+  opacity: 0.7;
+  outline: none;
+  pointer-events: none;
+  position: absolute;
+  top: 20%;
+  transition: opacity .2s;
+  -webkit-appearance: none;
+  -webkit-transition: .2s;
   width: 100%;
-}
\ No newline at end of file
+}
+
+ngeo-mapswipe .swipe-input:hover {
+  opacity: 1;
+}
+
+ngeo-mapswipe .swipe-input::-webkit-slider-thumb {
+  appearance: none;
+  background: #206020;
+  cursor: pointer;
+  height: 415px;
```

Comment:
height should be 100% (not tested, but normally it should follow the container).